### ISSUE TITLE
Allegro Ceny a udział oferty w programie

### ISSUE THREAD

**Issue by rtnet-pl** [allegro/allegro-api#5735]
Cześć, mam pytanie dot. programu Allegro Ceny.
W jaki sposób mogę sprawdzić czy dana oferta ma aktualnie przypisany znacznik Allegro Ceny? Czy to jest widoczne w /sale/badges?
Czy możecie włączyć mi ten program na sandbox dla konta 43950914 (rtnet)?

**Comment by AureliuszBrussy** [comment]
Tak, oznaczenie Allegro Cena jest widoczne w response dla [GET /sale/badges](https://developer.allegro.pl/documentation/#operation/getBadges). ID kampanii to "SUBSIDY", nazwa "Allegro Cena". Zgłosiłem prośbę oznaczenia oferty na Twoim koncie do zespołu odpowiedzialnego za usługę. Wrócę z informacją.

**Comment by AureliuszBrussy** [comment]
@rtnet-pl dodaliśmy ofertę 7689618179 do kampanii Allegro Ceny.

**Comment by rtnet-pl** [comment]
Dziękuję!

**Comment by AureliuszBrussy** [comment]
@rtnet-pl to jest efekt tego, że ręcznie uruchomiliśmy oznaczenie dla tej oferty - na środowisku produkcyjnym taka sytuacja nie będzie mieć miejsca.

**Comment by rtnet-pl** [comment]
OK, dzięki
